### DISCUSSION TITLE
[A-L] (9 Settembre 2021) Esercizio 1

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#238] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/152788993-47901586-0877-49a8-8c50-e1aafc6453a8.png)

**Comment by matypist** [comment]
_Simile ma non uguale a #154_

**Comment by dani3390** [comment]
![IMG_20220628_004140](https://user-images.githubusercontent.com/23482309/176048288-2c2197ba-8483-4452-b884-7d4d401730af.jpg)
